src/UI Component/User/paymenthistory.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+const PaymentHistory = () => {
+  // Dummy data for loan repayment history
+  const paymentHistory = [
+    { date: '2023-01-01', amount: 1000,  },
+    { date: '2023-02-01', amount: 1200 },
+    { date: '2023-03-01', amount: 800 },
+    // Add more entries as needed
+  ];
+
+  return (
+    <div className="container h-screen text-black mx-auto mt-10 overflow-x-auto">
+      <h1 className="text-3xl font-bold mb-6">Payment History</h1>
+      <div className="min-w-full overflow-hidden border border-black">
+        <table className="min-w-full">
+          <thead>
+            <tr className="grid grid-cols-2 gap-2 font-bold border border-black">
+              <th className="col-span-1 py-2 px-4 text-left">Date</th>
+              <th className="col-span-1 py-2 px-4 text-left">Amount</th>
+            </tr>
+          </thead>
+          <tbody>
+            {paymentHistory.map((entry, index) => (
```

Comment:
Please do not use index as key. Ideally, the paymentHistory array will come with an id from the backend. so simulate that here and use the id as key. 

Using INDEX as key is not advised as the array can change and this will cause an issue with rendering in react - performance-wise